Src/Base/AMReX_Math.H

```diff
@@ -225,6 +226,70 @@ std::uint64_t umulhi (std::uint64_t a, std::uint64_t b)
 }
 #endif
 
+template <typename T>
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+T comp_ellint_1 (T m)
+{
+    // Computing K based on DLMF
+    // https://dlmf.nist.gov/19.8
+    T tol = 1e-12;
```

Comment:
What if T is float?

Reply:
I will add a check on it to adjust the tolerance for a float to 1e-6.

Reply:
This should be fixed now.